voice/methods/calls/putCallsCallIdRecording.md

```diff
@@ -0,0 +1,28 @@
+{% method %}
+## Pause or resume recording on an active Call
+Pause or resume recording on an active phone call.
+
+### Request URL
+<code class="post">PUT</code>`https://voice.bandwidth.com/api/v2/accounts/{accountId}/calls/{callId}/recording`
+
+---
+
+### Supported Parameters
+| Parameter       | Description                                                                                                                            | Mandatory |
+|:----------------|:---------------------------------------------------------------------------------------------------------------------------------------|:----------|
+| state           | The recording state. Possible values: <br><br> `PAUSED` to pause an active recording<br>`RECORDING` to resume a paused recording<br>   | Yes       |
+{% common %}
+
+### Example: Pause a currently active recording
+{% sample lang="bash" %}
+```bash
+curl -u username:password -v -X PUT https://voice.bandwidth.com/api/v2/accounts/{accountId}/calls/{callId}/recording \
+	-H "Content-type: application/json" \
+	-d \
+	'
+	{
+		"state":"PAUSED"
```

Comment:
@mmeyers-bw, @mhossomi pointed out that this is uppercase although everything else in the api is lowercase.
Our code accepts both since we use `equalsIgnoreCase`.
Is it ok to change the doc to be lowercase?

Reply:
yeah, if we handle both, let's keep it consistent in the docs.